chcemvediet/locale/sk/LC_MESSAGES/django.po

```diff
@@ -1323,12 +1323,6 @@ msgstr "Prihlásiť pomocou Facebooku"
 msgid "account:social_buttons:google"
 msgstr "Prihlásiť pomocou Googlu"
 
-msgid "account:social_buttons:twitter"
-msgstr "Prihlásiť pomocou Twittra"
-
-msgid "account:social_buttons:linkedin"
```

Comment:
@martinmacko47 V subore `media/pages/sk/o-projekte` je link na twitter `https://twitter.com/chcemvedietsk` a v subore `media/pages/sk/faq` je `https://twitter.com/chcemvedietSK`. Oba linky su funkcne. Opravit alebo nechat tak ako to je?

Reply:
@viliambalaz Ano, tieto linky na twitter account nechaj. Mazeme iba login. Cize ikonku treba nechat, kedze sa pouziva tam. Fatba sa vsak tusim nikde nepouziva, ci? Tu by som teda vyhodil. Ked ju budeme potrebovat vieme ju vylovit z git historie a vratit.

Reply:
@martinmacko47 farba sa pouziva v `/sk/o-projekte/` v sekcii kontakt ked spravis 'hover' na twitter link.

Reply:
Jaj, jasne. Potom samozrejme farbu ponechat.